dulwich/porcelain.py

```diff
@@ -2222,7 +2223,10 @@ def describe(repo, abbrev=7):
 
         # If there are no tags, return the current commit
         if len(sorted_tags) == 0:
-            return f"g{find_unique_abbrev(r.object_store, r[r.head()].id)}"
+            object_id = r[r.head()].id
```

Comment:
Optional extra: we could relocate and re-use the `latest_commit` variable instead of creating `object_id` here.  That might make the code marginally more understandable, too.